requirements.txt

```diff
@@ -1,12 +1,15 @@
 # Requirements for asl-ml-immersion repository
-google-cloud-aiplatform==1.43.0
-google-cloud-pipeline-components==2.8.0
+
+# Google Cloud
+google-genai==0.2.2 # tentative version
```

Comment:
Latest version as of 2025/12/16. Will update later depending on the timing to merge the PR.